test/ketherhomepage.js

```diff
@@ -1,366 +1,200 @@
-const KetherHomepage = artifacts.require("./KetherHomepage.sol");
-// TODO should I query the contract to make sure the above values are right?
-const weiPixelPrice = web3.toWei(1000/1000000, "ether");
-const pixelsPerCell = 100;
-
-const oneHundredCellPrice = 10 * 10 * pixelsPerCell * weiPixelPrice;
-
-contract('KetherHomepage', function(accounts) {
-  const owner = accounts[0]; // this is the account we deploy as owner, see 2_deploy_contracts.js
-  const withdrawWallet = accounts[1];
-  const account1 = accounts[2];
-  const account2 = accounts[3];
-
-  web3.eth.sendTransaction({from:owner, to:account1, value: 10000000000000000000 }, function(err, r) { /* NOP */ });
-
-  it("should have correct constants by default", function() {
-    let KH;
-    return KetherHomepage.new(owner, withdrawWallet)
-      .then(function(instance) {
-        KH = instance;
-
-        return KH.weiPixelPrice.call();
-      })
-      .then(function(wpp) {
-        assert.equal(weiPixelPrice, wpp.toNumber());
-        return KH.pixelsPerCell.call();
-      })
-      .then(function(ppc) {
-        assert.equal(pixelsPerCell, ppc.toNumber());
-      });
+const { expect } = require('chai');
+
```

Comment:
Weeeeeeeeeeeeee

This whole thing was quite a PITA but it does look a fair bit nicer.

Also except(...).to.equal(...) are backwards because that was the easier vim macro to write. 🙃